src/main/resources/application.properties

```diff
@@ -18,6 +18,7 @@ management.endpoints.web.exposure.include=mappings
 springfox.documentation.swagger.v2.path=/api/docs
 spring.jpa.hibernate.ddl-auto=update
 app.admin.emails=${ADMIN_EMAILS:${env.ADMIN_EMAILS:phtcon@ucsb.edu}}
+app.sourceRepo=${SOURCE_REPO:${SystemInfo.java.sourceRepo}}
```

Comment:
This should be something like the following:

```
app.sourceRepo=${SOURCE_REPO:${env.SOURCE_REPO:ucsb-cs156/proj-happycows}}
```

What you have, i.e. `SystemInfo.java.sourceRepo` doesn't have any meaning here.   You need to model it after what's on line 20.  

The meaning is: 
* use the value of SOURCE_REPO if present
* If not use the value of env.SOURCE_REPO if present
* If not, use the default value `ucsb-cs156/proj-happycows`

Then, by setting SOURCE_REPO in your `.env` file and your Heroku Config vars, you can override the default.

Reply:
Change has been implemented.